src/client/pages/OfferNew/Checkout/InsuranceSummaryDetails.test.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  noCombo as mockedOfferDataNoCombo,
+  seApartementBrf as mockedOfferDataSeApartment,
+  dkHomeContentAccidentTravel as mockedOfferDataDkHomeAccidentTravel,
+} from 'utils/testData/offerDataMock'
+import { getAddress } from './InsuranceSummaryDetails'
+
+describe('getAddress function', () => {
+  describe('with Swedish quote', () => {
+    it('only returns street name and street number when quote is Swedish', () => {
+      const swedishApartmentQuoteDetails =
+        mockedOfferDataSeApartment.quotes[0].quoteDetails
+
+      const address = getAddress(swedishApartmentQuoteDetails)
+      expect(address).toBe('Testgatan 23')
+    })
+  })
+
+  describe('with Norwegian quotes', () => {
+    it('only returns street name and street number when quote is Norwegian Home Contents quote', () => {
+      const norwegianHomeContentsQuote = mockedOfferDataNoCombo.quotes[1]
+
+      const norwegianHomeContentsQuoteDetails =
+        norwegianHomeContentsQuote.quoteDetails
+
+      const address = getAddress(norwegianHomeContentsQuoteDetails)
+      expect(address).toBe('Testveien 23')
+    })
+
+    it('returns null when quote is Norwegian Travel quote', () => {
+      const norwegianTravelQuote = mockedOfferDataNoCombo.quotes[0]
+
+      const norwegianTravelQuoteDetails = norwegianTravelQuote.quoteDetails
+
+      const address = getAddress(norwegianTravelQuoteDetails)
+      expect(address).toBe(null)
+    })
```

Comment:
Do we call this function even for travel quote? 😅

Reply:
No we don't but since TS doesn't understand that I added a check that it just returns `null` it the quote lacks address and thought it was best to add a check for that as well so I didn't mess up 🙈

Reply:
Right, I understand. So the other way would be to place an "assert" statement at the top of the function. But this is probably "safer".

Reply:
Aha, don't know about the `assert` - will check that out. But yes maybe in this case keep it? But I agree it looks weird.

Reply:
Haha I was thinking about Python 🙃. It's strange that JS doesn't have assertions -- but I wrote a comment in the `getAddress` function about doing the same thing in the JS way :)

Reply:
Haha OK 😄 But sure, I'll throw an `Error` instead, that's probably better